### ISSUE TITLE
[1.12.2] Crash when remake or make a world(sometimes)

### ISSUE THREAD

**Issue by quentin452** [quentin452/1.7.10-Forge-renewed-v1#193]
edit : caused by optifine apparently

[crash-2023-01-22_18.42.47-server.txt](https://github.com/quentin452/privates-minecraft-modpack/files/10475086/crash-2023-01-22_18.42.47-server.txt)

**Comment by quentin452** [comment]
probably duplicated #190

**Comment by quentin452** [comment]
fixed by deleting DimensionalDoors-1.12.2-3.0.12

**Comment by quentin452** [comment]
```
java.lang.NullPointerException
    at net.minecraft.world.WorldEntitySpawner.performWorldGenSpawning(WorldEntitySpawner.java:428)
    at net.minecraft.world.gen.ChunkGeneratorOverworld.populate(ChunkGeneratorOverworld.java:466)
    at net.minecraft.world.chunk.Chunk.populate(Chunk.java:1019)
    at net.minecraft.world.chunk.Chunk.populate(Chunk.java:999)
    at net.minecraft.world.gen.ChunkProviderServer.provideChunk(ChunkProviderServer.java:157)
    at net.minecraft.server.MinecraftServer.initialWorldChunkLoad(MinecraftServer.java:344)
    at net.minecraft.server.integrated.IntegratedServer.loadAllWorlds(IntegratedServer.java:210)
    at net.minecraft.server.integrated.IntegratedServer.init(IntegratedServer.java:233)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:486)
    at java.lang.Thread.run(Thread.java:826)
    ````
    caused by optifine apparently

**Comment by quentin452** [comment]
[crash-2023-01-22_23.36.45-server.txt](https://github.com/quentin452/privates-minecraft-modpack/files/10475687/crash-2023-01-22_23.36.45-server.txt)

**Comment by quentin452** [comment]
fixed in V6.0modifiedbeta5servernotsupported by removing optifine
